### ISSUE TITLE
[Feature Request] Touch scrolling

### ISSUE THREAD

**Issue by Kreach37** [Suwayomi/Tachidesk-JUI#54]
---

## What feature should be added to Tachidesk-JUI?
Touch scrolling for touch screen user. For the reader and also in the chapter list page.
Just like in sorayomi.

**Comment by Robonau** [comment]
isn't this already a thing kinda
its not as fleshed out as it is in yomi but tapping defo scrolls

**Comment by Kreach37** [comment]
yeah but you can't scroll normally, like when you hold your finger and the page move with it. Just like every mobile app in existence. And the chapter list page is really awkward to have to use the side scrolling bar since no touch scrolling work there.

**Comment by Robonau** [comment]
ahh, u mean swipe scrolling?

**Comment by Kreach37** [comment]
Yeah I guess that's the proper term
